### ISSUE TITLE
qx compile does not complain about non existing qx classes

### ISSUE THREAD

**Issue by oetiker** [qooxdoo/qooxdoo-compiler#27]
`qx compile` does not seem to notice missing qx classes. It does complain about non existing symbols in other namespaces:

```javascript
var x = kk.yyy;
var w  = new qx.ui.form.ButtonXXX("hello");
```

```
qx compile 
2017-08-30 21:48:18.388 sponsor.ui.Popup: [346,28] Unresolved use of symbol kk.yyy
2017-08-30 21:48:18.798 [info ] makers          Writing target Source Target: source-output/ application sponsor
```

**Comment by johnspackman** [comment]
fixed
